src/test/java/uk/gov/hmcts/reform/civil/sampledata/CaseDataBuilder.java

```diff
@@ -2032,11 +2035,11 @@ public CaseDataBuilder atStateSdoFastTrackTrial() {
 
     public CaseDataBuilder atStateSdoDisposal() {
         disposalOrderWithoutHearing = DisposalOrderWithoutHearing.builder()
-             .input(String.format(
-            "Each party has the right to apply to have this Order set "
-                + "aside or varied. Any such application must be received "
-                + "by the Court (together with the appropriate fee) "
-                + "by 4pm on %s.", LocalDate.parse("2022-01-30")))
+            .input(String.format(
```

Comment:
Can you check all these space added, that is because the code is not related with our ticket

Reply:
Checked, the changes is not make from our side but it shows here, tried the gradle build and no error comes out. The conversation will marked as resolved